packages/react/src/Heading/Heading.featires.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
```

Comment:
typo in file name, would mess with automated examples in docs

```diff
- Heading.featires.stories.tsx
+ Heading.features.stories.tsx
```

Reply:
Tire 😄 thanks!